kernel/build/config-arm64

```diff
@@ -5198,8 +5378,30 @@ CONFIG_DRM_GEM_SHMEM_HELPER=y
 # end of ARM devices
 
 # CONFIG_DRM_RADEON is not set
-# CONFIG_DRM_AMDGPU is not set
+CONFIG_DRM_AMDGPU=m
```

Comment:
we are not using it, but it is required to enable drm_ttm_helper which is in turn required to build NVIDIA on adm64

Reply:
`drm_ttm_helper` was a dependency of i915 and amdgpu

Reply:
yes, but no i915 for arm64, so I enabled AMDGPU

Reply:
ahh I see, weird indeed